### PR TITLE
Attempt to add in map search bar

### PR DESCRIPTION
**Pull request by kaohman**
- Attempt to use react-leaflet-search to add a search bar to the map.
- Attempt to install different versions of React and Leaflet to fix map render bug.
- Also add main.scss file that was deleted by accident.
- See BikeMap.js for comments.

Issues:
- Unrelated to the search bar, map tilelayer is not loading correctly on refresh, only half the map loads and the position is incorrect. When you resize the screen, it loads correctly.
- Receiving "Cannot read property 'displayName' of undefined" error when trying to load the page with the search bar as a child component of Map in BikeMap.

@milkman4

### CODE REVIEW

File: src/Containers/BikeMap/BikeMap.js [hunk 1]
```diff
@@ -52,6 +54,15 @@ export class BikeMap extends Component {
             center={[30, 0]}
             zoom='3'>
             {this.markersToShow()}
+            {/* MapSearch is my withLeaflet wrapper on ReactLeafletZoomIndicator - I also tried wrapping it directly in this file */}
+            <MapSearch />
```
Comment (kaohman) [inline src/Containers/BikeMap/BikeMap.js:58]: MapSearch is my withLeaflet wrapper on ReactLeafletZoomIndicator - I also tried wrapping it directly in this file

File: src/Containers/BikeMap/BikeMap.js [hunk 2]
```diff
@@ -52,6 +54,15 @@ export class BikeMap extends Component {
...
+            {/* ReactLeaflet search is the search bar component - see BikeMap.scss for styling I added */}
+            <ReactLeafletSearch
```
Comment (kaohman) [inline src/Containers/BikeMap/BikeMap.js:60]: ReactLeaflet search is the search bar component - see BikeMap.scss for styling I added

File: src/Containers/BikeMap/BikeMap.js [hunk 3]
```diff
@@ -37,10 +39,10 @@ export class BikeMap extends Component {
   }
 
   render () {
-    // This setTimeout is needed in order for map to render tiles properly. Need css modules to load first.
-    setTimeout(() => {
-      this.setState({ loading: false })
-    }, 100);
+    // This setTimeout is needed in order for map to render tiles properly. Found this suggested fix online - something about needing css modules to load first. It slows down the map a ton though.
+    // setTimeout(() => {
```
Comment (kaohman) [inline src/Containers/BikeMap/BikeMap.js:43]: This setTimeout is needed in order for map to render tiles properly. Found this suggested fix online - something about needing css modules to load first. It slows down the map a ton though.

Comment (milkman4) [reply src/Containers/BikeMap/BikeMap.js:43]: I would put that in componentDidMount, not render - that way it only causes that delay once on the initial render.

File: src/Containers/BikeMap/BikeMap.js (hunk 1 above)
Comment (milkman4) [reply src/Containers/BikeMap/BikeMap.js:58]: ReactLeafletZoomIndicator is undefined, since it's not part of the leaflet search react library

**Pull request merged**